Sources/PlaybackSDK/PlayBackSDKManager.swift

```diff
@@ -407,10 +407,13 @@ public class PlaybackSDKManager {
         }
         
         // Adding extra details
+        sourceConfig.metadata["videoTitle"] = details.title
```

Comment:
Adding the `videoTitle` into the `sourceConfig.metadata` here is unused. I see you don't retrieve that information ahead.